src/env.h

```diff
@@ -92,13 +92,22 @@ struct PackageConfig {
   enum class Exists { Yes, No };
   enum class IsValid { Yes, No };
   enum class HasMain { Yes, No };
+  enum class HasExports { Yes, No };
   enum PackageType : uint32_t { None = 0, CommonJS, Module };
 
+  inline PackageConfig(Exists exists, IsValid is_valid, HasMain has_main,
+                       const std::string& main, PackageType type,
+                       HasExports has_exports, v8::Global<v8::Value>&& exports);
+  inline PackageConfig(PackageConfig&& pkg);
```

Comment:
I’m not sure if you’ve tried it, but if you can get away with 

```suggestion
  PackageConfig(PackageConfig&& pkg) = default;
```

here then I’d do that.

Reply:
I *think* I tried this initially but it still failed but that was a couple of changes ago so I'll definitely double-check. Thanks!

Reply:
Double-checked, running into the deleted constructor with the default implementation (`Global(const Global&) = delete`).

Reply:
@jkrems Yeah, but if you no longer declare `exports` as a const member I think this would go away, plus you can get rid of the invalid `const_cast`s (I didn’t realize before that the casts were necessary because `exports` is defined const, sorry – it’s generally not allowed to cast away constness with the intention of making an object modifiable “by force”).

Reply:
Oh! Let me try what that changes. Maybe that lets us undo a lot of this. 🤞 Thanks for the hint!

Reply:
Ooooh yesssss. Removing `const` from the declaration made all the defaults work which removes the need for a bunch of code to implement custom (and potentially invalid) constructors. Undid the `env-inl.h` changes.